Receipty iOS app FINAL

muhammadanas0716 opened the pull request.

faisalsayed10 commented:
@muhammadanas0716 for the PR, you dont have to upload your app here. we just need the README.md file as described here: https://github.com/hackclub/cider/blob/main/README.md

faisalsayed10 reviewed (changes requested):
I reviewed your app, I really love the idea. I didn't really have many feedback for your app, it already looks pretty good. I left two-three few minor improvement requests, lmk once u get done w it, i'll re-review and we can merge this :))

muhammadanas0716 commented:
removed the project files

faisalsayed10 commented:
Please delete these 3 files from the PR
<img width="932" alt="image" src="https://github.com/user-attachments/assets/486c4dae-3750-49d7-98ea-6d3200309468">

faisalsayed10 commented:
looks good to me, ready to approve after the pr is fixed

muhammadanas0716 commented:
![image](https://github.com/user-attachments/assets/c948ad9b-0429-47ea-8706-c4b6f2cbe9df)

Deleted both of DS store files but that readme.md (the one you asked me to delete) has already been deleted. I deleted prev.

faisalsayed10 commented:
yes, can u please revert your changes for the submissions file?

faisalsayed10 commented:
Have you completed ID verification btw?

muhammadanas0716 commented:
ID verification? Nope

muhammadanas0716 commented:
Oh yes sorry, I did buy something with my tickets and yes I did get approved - this one right?
![image](https://github.com/user-attachments/assets/73b28289-fdfa-4bf9-a29a-6c7375c81b32)

faisalsayed10 commented:
Thanks, that's what I needed
